core-java-modules/core-java-collections-array-list-2/README.md

```diff
@@ -0,0 +1,6 @@
+## Core Java Collections ArrayList
+
+This module contains articles about the Java ArrayList collection
+
+### Relevant Articles: 
+- [Create an ArrayList with Multiple Object Types](https://drafts.baeldung.com/create-an-arraylist-with-multiple-object-types/)
```

Comment:
Let's omit this line until after the article goes live. Your editor will add it afterward with the final URL.